com.unity.netcode.adapter.utp/Runtime/UnityTransport.cs

```diff
@@ -463,12 +461,21 @@ public override void DisconnectRemoteClient(ulong clientId)
                 }
 
                 // we need to cleanup any SendQueues for this connectionID;
-                var keys = m_SendQueue.Keys.Where(k => k.ClientId == clientId).ToList();
+                var keys = new NativeList<SendTarget>(16, Allocator.Temp); // use nativelist and manual foreach to avoid allocations
```

Comment:
@Cosmin-B what do you think should we do this or keep linq here?

Reply:
Yes. Also, is worth noting that this call would not happen frequently at all... so the performance benefit is super tiny.